.gitignore

```diff
@@ -1,3 +1,6 @@
+.idea
+.vscode
```

Comment:
Added these two lines to the `.gitignore` so that new users are not prompted to add IDE related files.